tech.json

```diff
@@ -156,14 +163,39 @@
         },
         {
           "name": "canMidAirMorph",
-          "requires": ["Morph"],
+          "requires": [ "Morph" ],
           "note": [
             "The ability to enter Morph Ball while in the air.",
             "There can be an expectation to Morph quickly as well, usually following a jump from the ground.",
             "Out of water physics this means jumping and morphing in a 4 tile high space.",
             "In water physics this means jumping and morphing in a 3 tile high space (Samus takes up 3 tiles when standing)."
           ],
           "extensionTechs": [
+            {
+              "name": "canWallJumpIntoMorph",
```

Comment:
I was thinking `canWallJumpInstantMorph`
The name and description still sound like this is nothing special.

Pressing down right away is pretty important.  Turning around would break the instant morph effect and you'd need two down presses again.